crates/context/src/lib.rs

```diff
@@ -307,11 +307,18 @@ impl ContextManager {
         blob_id: calimero_primitives::blobs::BlobId,
         source: calimero_primitives::application::ApplicationSource,
         version: Option<semver::Version>,
+        contract_app_id: Option<String>,
     ) -> eyre::Result<calimero_primitives::application::ApplicationId> {
+        let contract_app_id: Box<str> = match contract_app_id {
+            Some(s) => s.into_boxed_str(),
+            None => Box::from(""),
```

Comment:
When can contract_app_id be empty?

Reply:
When installing local application which has no metadata saved in the smart contract package manager